.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,76 @@
+name: Build project, then Docker Image and publish to Github Artifact Registry
+
+on:
+  push:
+    # Publish semver tags as releases.
+    tags: ['v*.*.*']
```

Comment:
How are tags created? Is it in the workflow or for now we do it manually?

Reply:
Manually for now